packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -181,7 +181,9 @@ export default defineComponent({
     },
 
     showActions() {
```

Comment:
I was most likely involved in settings this up, but having a v-if on the wrapping div in a component seems like an antipattern versus having the parent component define whether the child should get rendered at all (I may miss the whole upload logic being handled here, maybe that needs to happen individually?)

Reply:
I know 😞 Hence my note in the initial post, this is the "Easy-but-not-so-nice"-solution. Moving the upload logic is probably nicer but also way more complicated. But if we all decide to go this way, I'll rework this PR.